backend/app/services/task_dispatcher.py

```
@@ -368,9 +371,32 @@ def schedule_dispatch(self, task_id: int) -> None:
+                asyncio.set_event_loop(new_loop)
+                try:
+                    return new_loop.run_until_complete(
+                        self._dispatch_task_async(task_id)
+                    )
+                finally:
+                    new_loop.close()
+
+            # Run in thread pool to avoid blocking
+            with concurrent.futures.ThreadPoolExecutor() as executor:
+                future = executor.submit(run_async_in_thread)
+                result = future.result(timeout=30)
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)